bin/main.rb

```diff
@@ -0,0 +1,24 @@
+puts 'welcome to Tic Tac Toe'
+puts 'Enter your Name (First Player): '
+player_one = gets.chomp
+# save the player_one name in this var
+puts "#{player_one} your symbol is X "
+
+puts 'Enter your Name (Second Player): '
+player_two = gets.chomp
+# save the player_one name in this var
+puts "#{player_two} your symbol is O "
+
+puts '-------------'
+puts '| a | b | c |'
+puts '| d | e | f |'
+puts '| g | h | i |'
+puts '-------------'
```

Comment:
- [ ] Kindly use a loop (can be any loop but I recommend you use a while loop for scalability) to mock the users' taking turns. It doesn't have to work yet but it would be nice to see that you know it will be included in the program.

Remember to tell the users:
- [ ] which player has the turn
- [ ] what are the spaces available
- [ ] whether or not a move is valid

Remember, it does NOT have to work yet, but it would be good to see how you think to do so.